src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -324,15 +325,34 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
 
                     throw new InvalidOperationException(message);
                 }
+                restoreAuditProperties = new();
```

Comment:
Would this dictionary need to be case-insensitive?
```suggestion
                restoreAuditProperties = new(StringComparer.OrdinalIgnoreCase);
```

Reply:
It needs the path aware version.